v1/summary.js

```diff
@@ -25,6 +25,16 @@ module.exports = (options) => ({
             if (terms && terms.description && terms.description.length) {
                 return terms.description[0];
             }
+        },
+        processCoords(coords) {
+            if (!coords || !coords.length) {
+                return undefined;
+            }
+
+            const coord = coords[0];
+            delete coord.primary;
+            delete coord.globe;
+            return coord;
```

Comment:
Since `lat` and `lon` are `requiredProperties` should we double-check here we have them and return `undefined` if we don't? It only seems prudent.